src/lib/http/index.ts

```diff
@@ -2,16 +2,20 @@ import fetch from './fetch';
 
 type Params = { [key: string]: any };
 
+type HttpConfig = { credentials?: RequestCredentials; headers?: any; addBracketsToArrayValues?: boolean };
+
+const httpConfigParams = ['credentials', 'headers', 'addBracketsToArrayValues'];
```

Comment:
this should be typed... and honestly, i dont love how you're figuring out this is a config object. `addBracketsToArrayValues` smells like a workaround that is hiding a bug, and to make this work we have to pass it in all the time.